assets/javascript/app.js

```diff
@@ -1,43 +1,86 @@
-//Define variables
-//Create variable to hold user input. This is the search term the user enters in the main recipe search field.
-var userInput = "";
-
-//Initialize Firebase here.
-
-//Create database variable to create reference to firebase.database().
-	//var database = firebase.database();
-
-
-//Click event for find/submit button for main recipe search field.
-$("#submit").on("click", function() {
-	//Prevent form from submitting itself.
-	event.preventDefault();
-
-	//Grab the search term that the user enters in the main recipe search field.
-	userInput = $("#user-input").val().trim().toLowerCase();
-	//print userInput value to console for debugging purposes.
-	console.log("User's recipe search term: " + userInput);
-
-	//Construct our query URL here to access and obtain data from the EDAMAM API.
-	//var appID = "";
-	//var appKey = "";
-	//var queryURL = ""
-	
-	//Our jQuery AJAX method. Perform AJAX GET request to the queryURL to get data from the EDAMAM API.
-   	//$.ajax({
-        //url: queryURL,
-        //method: "GET"
-      //})
-
-      //After the data from the AJAX request comes back.
-      //.done(function(response) {
-      	//log the API response to the console for debugging purposes.
-      	//console.log(response);
-      	//create variable to hold response data
-      	//var results = ;
-    //});
+// Initialize Firebase
+ var config = {
+	 apiKey: "AIzaSyCC1t5np8kPEV2yC0heaC4z0l6JtYNEMhY",
+	 authDomain: "projectlongshot-469ee.firebaseapp.com",
+	 databaseURL: "https://projectlongshot-469ee.firebaseio.com",
+	 projectId: "projectlongshot-469ee",
+	 storageBucket: "",
+	 messagingSenderId: "333236357647"
+ };
+ firebase.initializeApp(config);
+database = firebase.database();
+
+var queryURLbase = "https://api.edamam.com/search?&app_id=4a5d81a2&app_key=379308ab9da9a8ee47f63563d2774ac4&from=0&to9&q=";
+var userInput;
+
+var imgAPI;
+var label;
+var recipe;
+var sourceLink;
+function testAjax(queryURL) {
+   $.ajax({
+       url: queryURL,
+       method: 'GET'
+   }).done(function(data) {
+       // console.log(data);
+       // console.log(queryURL);
+       for(var i = 0; i<9; i++){
+var card = $("<div>");
+card.addClass("card");
+var cardImg =$("<div>");
+cardImg.addClass("card-image");
+var img = $("<img>");
+imgAPI = data.hits[i].recipe.image;
+img.attr("src",imgAPI);
+cardImg.append(img);
+card.append(cardImg);
+var cardContent = $("<div>");
+cardContent.addClass("card-content");
+var spanTitle = $("<span>");
+spanTitle.addClass("card-title");
+label = data.hits[i].recipe.label;
+spanTitle.text(label);
+var pRecipe = $("<p>");
+recipe =data.hits[i].recipe.ingredients[0].text;
```

Comment:
I'm wondering if we need to have both the ingredients and the recipe link? Why not just have the recipe link? It would help cut down on the amount of text on the page and help keep the cards the same size.